stable/rabbitmq-ha/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: rabbitmq-ha
 apiVersion: v1
 appVersion: 3.8.0
-version: 1.40.1
+version: 1.40.2
```

Comment:
Once again, this needs a minor version increase for new features. Please see semver.org

Reply:
Then what do I have to put? 1.41.0, correct?

Reply:
1.41.0 currently, yes. You also now need to fix the merge conflict.